spec/factories.rb

```diff
@@ -140,9 +140,10 @@ def r_str
   end
 
   factory(:location) do
-    address "unicorn city"
-    lat 1
-    lng 2
+    address "Starco Mart, Mission Street, West SoMa, San Francisco, San Francisco "\
+            "City and County, Kalifornien, 94103, Vereinigte Staaten von Amerika"
+    lat 37.78
+    lng -122.41
```

Comment:
Ambiguous negative number operator. Parenthesize the method arguments if it's surely a negative number operator, or add a whitespace to the right of the `-` if it should be a subtraction.